src/app/modules/group/http-services/group-update.service.ts

```diff
@@ -0,0 +1,29 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { appConfig } from 'src/app/shared/helpers/config';
+import { assertSuccess, SimpleActionResponse } from 'src/app/shared/http-services/action-response';
+
+export interface GroupChanges {
+  name?: string,
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GroupUpdateService {
+
+  constructor(private http: HttpClient) {
+  }
+
+  updateGroup(
+    groupId: string,
+    changes: GroupChanges
+  ) : Observable<void> {
+    return this.http.put<SimpleActionResponse>(
+      `${appConfig().apiUrl}/groups/${groupId}`,
+      changes,
+    ).pipe(map(assertSuccess));
```

Comment:
Lots of line break, I don't think it makes things very readable (of course that's subjective):
```suggestion
  updateGroup(groupId: string, changes: GroupChanges) : Observable<void> {
    return this.http.put<SimpleActionResponse>(`${appConfig().apiUrl}/groups/${groupId}`, changes).pipe(
      map(assertSuccess),
    );
```

Reply:
fixed